persistence/rest/src/main/resources/schema/infinispan-cachestore-rest-config-10.1.xsd

```diff
@@ -16,15 +16,9 @@
           <xs:element name="remote-server" type="config:remote-server" minOccurs="1" maxOccurs="unbounded"/>
           <xs:element name="connection-pool" type="tns:rest-connection-pool" minOccurs="1" maxOccurs="unbounded"/>
         </xs:sequence>
-
-        <xs:attribute name="append-cache-name-to-path" type="xs:boolean" default="${RestStore.appendCacheNameToPath}">
-          <xs:annotation>
-            <xs:documentation>Whether to append the name of the cache to the path.</xs:documentation>
-          </xs:annotation>
-        </xs:attribute>
-        <xs:attribute name="path" type="xs:string" default="${RestStore.path}">
+        <xs:attribute name="cache-name" type="xs:string" default="default">
```

Comment:
```suggestion
        <xs:attribute name="cache-name" type="xs:string" default="${RestStore.cacheName}">
```

Reply:
As you suggested earlier, the default will come from the InitializationContext, so no need to changer this anymore